Holovibes/sources/thread/information_worker.cc

```diff
@@ -306,24 +309,29 @@ void InformationWorker::display_gui_information()
         }
     }
 
-    for (auto const& [key, value] : FastUpdatesMap::map<QueueType>)
+    if (api::get_import_type() != ImportType::None)
     {
-        if (key == QueueType::UNDEFINED)
-            continue;
-        auto currentLoad = std::get<0>(*value).load();
-        auto maxLoad = std::get<1>(*value).load();
-
-        to_display << "<tr style=\"color:";
-        if (queue_type_to_string_.at(key) == "Output Queue")
-            to_display << "white";
-        else
-            to_display << get_load_color(currentLoad, maxLoad);
+        for (auto const& [key, value] : FastUpdatesMap::map<QueueType>)
+        {
+            if (key == QueueType::UNDEFINED)
+                continue;
+            auto currentLoad = std::get<0>(*value).load();
+            auto maxLoad = std::get<1>(*value).load();
+
+            to_display << "<tr style=\"color:";
+            if (key == QueueType::OUTPUT_QUEUE)
+                to_display << "white";
+            else if (key == QueueType::INPUT_QUEUE)
+                to_display << get_load_color(currentLoad, maxLoad, 0.3f, 0.8f);
```

Comment:
even if it s only once, use macros instead of magics values

Reply:
Added macros